<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appSettingsTab</class>
 <widget class="QWidget" name="appSettingsTab">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>767</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>418</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>418</width>
    <height>1200</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>415</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This panel specifies application settings currently in use. Click &amp;quot;Make default&amp;quot; if you want these settings to apply the next time you start VAPOR.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Miscellaneous Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="textLabel1_2_3">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Specify a value between 1 and 100 to control the image quality in a saved jpeg file.</string>
           </property>
           <property name="text">
            <string>JPEG quality (1-100)</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer2_2_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>69</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="jpegQualityEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>32767</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Specify a quality level of the JPEG files to be created</string>
           </property>
           <property name="text">
            <string>75</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Automatically save session file</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="autosaveCheckbox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Check to enable automatic session saving</string>
           </property>
           <property name="whatsThis">
            <string>If the session change interval is positive and if the automatic save checkbox is checked, then every time the specified number of session changes has been made the current session is saved to the session autosave file.  This is useful if there is a likelihood of crash or of terminating the application, and you would like to be able to recover your last session.</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Setting changes per save</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="autosaveIntervalEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>87</width>
             <height>32767</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Specify the number of session changes per auto save</string>
           </property>
           <property name="whatsThis">
            <string>If the session change interval is positive and if the automatic save checkbox is checked, then every time the specified number of session changes has been made the current session is saved to the session autosave file.  This is useful if there is a likelihood of crash or of terminating the application, and you would like to be able to recover your last session.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Progress Monitoring Files</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="_8">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1_2_4">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Log File Name</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer84">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>82</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="logFilePathButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Specify Log File Name</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="logFilePathEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Specify the log file name (on Unix,  specify &quot;stderr&quot; for logging to console)</string>
           </property>
           <property name="whatsThis">
            <string>The log file name will be used to write diagnostics, warnings and errors to file.  On unix platforms, if the log file name is &quot;stderr&quot; or if the environment variable &quot;VAPOR_DEBUG&quot; is set, then these messages will be written to the console.</string>
           </property>
           <property name="text">
            <string>/tmp</string>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_9">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1_2_5_2_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Name of file where sessions are automatically saved</string>
           </property>
           <property name="whatsThis">
            <string>The file specified for auto save is overwritten every time there is an automatic session save</string>
           </property>
           <property name="text">
            <string>Autosave file</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer85">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>82</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="autosavePathButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Click to choose session auto save filename</string>
           </property>
           <property name="whatsThis">
            <string>The file specified for auto save is overwritten every time there is an automatic session save</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="autosaveFilenameEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Specify the file name and path (if not in the current directory) where sessions are automatically saved</string>
           </property>
           <property name="whatsThis">
            <string>The file specified for auto save is overwritten every time there is an automatic session save</string>
           </property>
           <property name="text">
            <string>/tmp</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_10">
     <item>
      <spacer name="spacer205">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>71</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>32767</height>
        </size>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="defaultButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Click to reset all user preferences and parameter values to their original default values</string>
       </property>
       <property name="text">
        <string>Restore Defaults</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_3">
     <item>
      <layout class="QVBoxLayout" name="_4">
       <item>
        <widget class="QCheckBox" name="warnMissingDataCheckbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>When checked a warning message will be presented if data is unavailable</string>
         </property>
         <property name="text">
          <string>Show warning messages for missing data</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="lowerAccuracyCheckbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>If requested refinement or compression level is unavailable, use the highest available level, with no warning message</string>
         </property>
         <property name="text">
          <string>When data is missing, use less accurate data if available</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="trackMouseCheckbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Check to display position, color, opacity at mouse position in editor</string>
         </property>
         <property name="text">
          <string>Track mouse position in Transfer Function editors</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="noShowCitationCheckbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Don't display the reminder to cite VAPOR when an image is saved.</string>
         </property>
         <property name="text">
          <string>Do not display citation information</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="logfileCheckbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Check box to write errors, warnings, diagnostics, etc. to specified log file</string>
         </property>
         <property name="text">
          <string>Log error messages, etc. to file.</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer202">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>367</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
